src/index.js

```diff
@@ -1,13 +1,13 @@
 import React, { useState, useEffect } from 'react';
-import ReactDOM from 'react-dom';
-import qs from 'query-string';
+import * as ReactDOM from 'react-dom';
```

Comment:
Exactly why do we need this?

Reply:
did you mean the wildcard `*` import style or `ReactDOM` in general?

Reply:
Oh no it was a blunder from me. Sorry